dev/io.openliberty.jakarta.cdi.3.0/src/com/ibm/ws/cdi/impl/CDI30ExtensionSPIProvider.java

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.cdi.impl;
+
+import java.util.Map;
+
+import com.ibm.ws.cdi.CDIException;
+import com.ibm.ws.cdi.internal.interfaces.CDIUtils;
+import com.ibm.ws.cdi.internal.interfaces.CDIRuntime;
+import com.ibm.ws.cdi.internal.interfaces.ExtensionArchive;
+import com.ibm.ws.cdi.internal.interfaces.ExtensionArchiveProvider;
+import com.ibm.ws.cdi.internal.interfaces.WebSphereCDIDeployment;
+
+import io.openliberty.cdi.spi.CDIExtensionMetadata;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+import org.osgi.framework.ServiceReference;
+
+@Component(configurationPolicy = ConfigurationPolicy.IGNORE)
+public class CDI30ExtensionSPIProvider implements ExtensionArchiveProvider {
```

Comment:
This shouldn't go in the API bundle

Reply:
Turns out we don't need this class at all because the `com.ibm.ws.cdi.2.0.weld` bundle is also included in the CDI 3.0 feature.